src/main/webapp/app/exam/manage/exam-status.component.ts

```diff
@@ -0,0 +1,172 @@
+import { Component, Input, OnChanges } from '@angular/core';
+import { faArrowRight, faCalendarCheck, faCheckCircle, faTimes, faTimesCircle, faCalendarTimes, faDotCircle } from '@fortawesome/free-solid-svg-icons';
+import { Exam } from 'app/entities/exam.model';
+import { ExamChecklistService } from 'app/exam/manage/exams/exam-checklist-component/exam-checklist.service';
+import { ExamChecklist } from 'app/entities/exam-checklist.model';
+import dayjs from 'dayjs/esm';
+import { round } from 'app/shared/util/utils';
+
+export enum ExamReviewState {
+    UNSET = 'unset',
+    PLANNED = 'planned',
+    RUNNING = 'running',
+    FINISHED = 'finished',
+}
+
+export enum ExamConductionState {
+    PLANNED,
+    RUNNING,
+    FINISHED,
+}
+
+@Component({
+    selector: 'jhi-exam-status',
+    templateUrl: './exam-status.component.html',
+    styleUrls: ['./exam-status.component.scss'],
+})
+export class ExamStatusComponent implements OnChanges {
+    @Input()
+    public exam: Exam;
+
+    examChecklist: ExamChecklist;
+    numberOfGeneratedStudentExams: number;
+
+    configureExercises: boolean;
```

Comment:
Nit: I think `configuredExercises` would be a better name as the variable shows that the instructor has already successfully finished the associated task. Applies to the next three variables as well.

Reply:
I chose the variable names so that they refer to the steps displayed to the user. These steps are formulated as action items on purpose. But I agree that it might be confusing in the code. Changed in 7f41df2

Reply:
Thanks!